wolfcrypt/src/port/atmel/atmel.c

```diff
@@ -190,7 +190,7 @@ int wolfCrypt_ATECC_SetConfig(ATCAIfaceCfg* cfg)
     XMEMSET(&cfg_ateccx08a_i2c_pi, 0, sizeof(cfg_ateccx08a_i2c_pi));
     cfg_ateccx08a_i2c_pi.iface_type            = cfg->iface_type;
     cfg_ateccx08a_i2c_pi.devtype               = cfg->devtype;
-    cfg_ateccx08a_i2c_pi.atcai2c.slave_address = cfg->atcai2c.slave_address;
+    cfg_ateccx08a_i2c_pi.atcai2c.address       = cfg->atcai2c.address;
```

Comment:
Is there a way to make this change without breaking compatibility with older ATCA releases? Perhaps there is a macro or version to check for that?